src/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/Asn1/TbsCertificateAsn.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.Asn1;
+
+namespace System.Security.Cryptography.X509Certificates.Asn1
+{
+    // https://tools.ietf.org/html/rfc5280#section-4.1
+    //
+    // TBSCertificate  ::=  SEQUENCE  {
+    //     version         [0]  Version DEFAULT v1,
+    //     serialNumber         CertificateSerialNumber,
+    //     signature            AlgorithmIdentifier,
+    //     issuer               Name,
+    //     validity             Validity,
+    //     subject              Name,
+    //     subjectPublicKeyInfo SubjectPublicKeyInfo,
+    //     issuerUniqueID  [1]  IMPLICIT UniqueIdentifier OPTIONAL,
+    //                          -- If present, version MUST be v2 or v3
+    //     subjectUniqueID [2]  IMPLICIT UniqueIdentifier OPTIONAL,
+    //                          -- If present, version MUST be v2 or v3
+    //      extensions      [3]  Extensions OPTIONAL
```

Comment:
Super-nit: Looks like an extra space got pasted in before `extension`

Reply:
Removed.